<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuDialog</class>
 <widget class="QDialog" name="MenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Считалочка</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать счет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </widget>
  <widget class="QListView" name="participantsListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="countersListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="stdPresets">
     <property name="title">
      <string>Загрузить стандартные наборы</string>
     </property>
    </widget>
    <addaction name="openParticipants"/>
    <addaction name="openCountings"/>
    <addaction name="stdPresets"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QLabel" name="validatorLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <action name="openParticipants">
   <property name="text">
    <string>Открыть участников</string>
   </property>
  </action>
  <action name="openCountings">
   <property name="text">
    <string>Открыть считалочки</string>
   </property>
  </action>
  <action name="pm5group">
   <property name="text">
    <string>1 курс 5 группа</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
